### PR TITLE
[MODULAR] Blueshield locker stuff and slight vendor update

### PR DESCRIPTION
**Pull request by axietheaxolotl**
## About The Pull Request

Yo. BS no longer has cringe wooden closest, they now have cool blue metal locker, isn't that epic? Also, BS shiz has been added to vendor, so clothing in the locker's been nuked 'cause of muh bloat.

Also, mantles are in the vendor now, because why not.

## Why It's Good For The Game

muh bloat
muh wooden closet that makes no sense
muh mantles and muh vendor

## Changelog
:cl:
add: added actual locker for BS and added mantles to vendor
imageadd: added le icon for BS locker
/:cl:

### CODE REVIEW

**Comment by GoldenAlpharex** [comment]
A few little questions/requests for this PR or another one:

1. Could you add a picture of the new Blueshield locker so that we can give feedback on it?
2. Would it be possible that you return the jumpsuits (at least the default ones) to the lockers? NT Rep doesn't have jumpsuits in their locker anymore and it's kind of awkward to have to go all the way to the vendor to be able to get that normal NT Rep jumpsuit I wanted, because it's not in the locker anymore.
3. Do you think you could look into letting people insert their clothes back into the vendor? Say, for instance, NT Rep can't put a Nanotrasen Representative jumpskirt back in the vendor, and I got a feeling they're probably not the only ones, we'd have to look into the vendor code a bit more to confirm how to do this, though.
4. In general, would it be possible to perhaps lower the prices of the stuff in that locker? It just kinda feels like I'm getting scammed out of my money to be able to have a look I used to have for free. I guess this one is probably something we'd have to take longer to discuss overall, could move it to Discord.

File: modular_skyrat/modules/command_vendor/code/vending/vending.dm [hunk 1]
```diff
@@ -10,39 +10,47 @@
...
-		/obj/item/storage/backpack/duffelbag/captain = 1, /obj/item/clothing/shoes/sneakers/brown = 1
+		/obj/item/clothing/suit/captunic = 1, /obj/item/clothing/neck/mantle/capmantle = 1, /obj/item/storage/backpack/captain = 1,
+		/obj/item/storage/backpack/satchel/cap = 1, /obj/item/storage/backpack/duffelbag/captain = 1, /obj/item/clothing/shoes/sneakers/brown = 1,
+
+		// BLUESHIELD
+		/obj/item/clothing/head/beret/blueshield = 1, /obj/item/clothing/head/beret/blueshield/navy = 1,
+		/obj/item/clothing/under/rank/security/blueshield = 1, /obj/item/clothing/under/rank/security/blueshieldturtleneck = 1,
+		/obj/item/clothing/suit/armor/vest/blueshield = 1, /obj/item/clothing/neck/mantle/bsmantle = 1,
+		/obj/item/storage/backpack/blueshield = 1, /obj/item/storage/backpack/satchel/blueshield = 1,
+		/obj/item/storage/backpack/duffel/blueshield = 1, /obj/item/clothing/shoes/laceup = 1
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/command_vendor/code/vending/vending.dm]: If you want to split things on multiple lines, let's do it right all the way.
Put only one element per line here, I'd do a suggestion but GitHub isn't happy about letting us make suggestions on something that has removed lines, for some reason. This is something that should be done all throughout the file, that being said.

**Review by GoldenAlpharex — CHANGES_REQUESTED** [review]
You're almost there, just format your lists a bit better in `vending.dm` and this'll be good to go.

**Comment by axietheaxolotl** [comment]
### Blueshield Locker Icon
bslocker

File: modular_skyrat/modules/command_vendor/code/vending/vending.dm [hunk 2]
```diff
@@ -10,61 +10,123 @@
...
+		/obj/item/clothing/under/rank/captain/parade = 1,
+		/obj/item/clothing/suit/toggle/captains_parade = 1,
+		/obj/item/clothing/suit/captunic = 1,
+		/obj/item/clothing/neck/mantle/capmantle = 1,
+		/obj/item/storage/backpack/captain = 1,
+		/obj/item/storage/backpack/satchel/cap = 1,
+		/obj/item/storage/backpack/duffelbag/captain = 1,
+		/obj/item/clothing/shoes/sneakers/brown = 1,
+
+		// BLUESHIELD
```
Comment (FlamingLily) [inline modular_skyrat/modules/command_vendor/code/vending/vending.dm:32]: Does the Blueshield not have a unique access?

Comment (jjpark-kb) [reply modular_skyrat/modules/command_vendor/code/vending/vending.dm:32]: yea, this really needs to be fixed. If its read like this, this means captain can access blueshield stuff.

Comment (GoldenAlpharex) [reply modular_skyrat/modules/command_vendor/code/vending/vending.dm:32]: > yea, this really needs to be fixed. If its read like this, this means captain can access blueshield stuff.

I mean, to be fair, it's not inherently a bad thing.

**Comment by axietheaxolotl** [comment]
i will fix this soon:tm:

**Comment by GoldenAlpharex** [comment]
There's still a merge conflict @axietheaxolotl.

**Comment by axietheaxolotl** [comment]
yea, I know. Gimmie a sec and I'll fix it.

**Comment by axietheaxolotl** [comment]
reee

**Pull request merged**